### PR TITLE
ember-cli Meeting Notes - 2017-08-17

### PR DESCRIPTION
**Pull request by rwjblue**

### CODE REVIEW

File: ember-cli/2017-08/august-17.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# ember-cli Meeting Notes - 2017-08-17
+
+## Attendees
+
+## Agenda
```
Comment (rwjblue) [inline ember-cli/2017-08/august-17.md]: Please reply here to add agenda items

Comment (rwjblue) [inline ember-cli/2017-08/august-17.md]: Discuss using [@std/esm](https://github.com/standard-things/esm) in ember-cli (and possibly by default for addons). 

Good writeup in the introductory blog post: https://medium.com/web-on-the-edge/es-modules-in-node-today-32cff914e4b.

Comment (rwjblue) [inline ember-cli/2017-08/august-17.md]: Discuss and review with the team the result of @kratiahuja and @rwjblue's conversation on fastboot + engines + meta config.

Comment (locks) [inline ember-cli/2017-08/august-17.md]: @ember/string, ember-cli-babel, babel-plugin-ember-modules-api-polyfill, ??? (Ember.String prototype extension addon/babel plugin)

**Comment by hjdivad** [comment]
cc @rwjblue @hjdivad @locks @kellyselden @twokul @kratiahuja @kategengler

**Comment by kellyselden** [comment]
Thanks again @hjdivad

**Pull request merged**